django_select2/forms.py

```diff
@@ -371,6 +382,9 @@ def filter_queryset(self, term, queryset=None):
         for t in [t for t in term.split(' ') if not t == '']:
             select &= reduce(lambda x, y: x | Q(**{y: t}), search_fields,
                              Q(**{search_fields[0]: t}))
+        if kwargs:
+            select &= Q(**kwargs)
```

Comment:
This assumes you want to `and` the query.
This should be easily overwritten.

Maybe you can make a docstring that explains to pop kwargs before calling super, if you want to implement your own select.